Source/CesiumRuntime/Private/CesiumPrimitive.h

```diff
@@ -101,6 +101,19 @@ class CesiumPrimitiveData {
 
   std::optional<Cesium3DTilesSelection::BoundingVolume> boundingVolume;
 
+  /**
+   * The factor by which the positions in the glTF primitive is scaled up when
+   * the Unreal mesh is populated.
+   *
+   * We scale up the meshes because Chaos has a degenerate triangle epsilon test
+   * in `TriangleMeshImplicitObject.cpp` that is almost laughably too eager.
+   * Perhaps it would be fine if our meshes actually used units of centimeters
+   * like UE, but they usually use meters instead. With a factor of 1.0, UE will
+   * consider a right triangle that is slightly less than ~10cm on each side to
+   * be degenerate.
+   */
+  static constexpr double positionScaleFactor = 1024.0;
```

Comment:
Would be nice to see a comment as to why 1024 was chosen. From the PR you wrote "chosen so that only the exponent of the floating point number is affected, not the mantissa"